src/frontends/paddle/src/op/partial_ops.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <map>
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+
+template <typename T>
```

Comment:
Can we use this ```template``` you defined to connect these 2 ops ?  which may help to enable more partial like ops.

Reply:
I have tried some methods but failed. 
`Add` op requires two `Node` as inputs, but `Concat` op requires a `NodeVector` and a integer to specify operate on which axis